pbkhrv/rtl_433-hass-addons
# Support running from stand-alone docker

## Summary
Support running the auto-discovery portion of this from a stand-alone docker container without relying on the add-ons. README is updated to explain how this can be run directly in docker as well. This addresses #76 

## Alternatives Considered
Starting a new repo that does the same thing as this repo, just without reliance on add-ons.

## Testing Steps
Running the following directly in Docker should allow the project to work similarly to the add-on.

```
docker run -e MQTT_HOST=mqtt.host.com -e MQTT_USERNAME=username -e MQTT_PASSWORD=password ghcr.io/pbkhrv/rtl_433-hass-addons-rtl_433_mqtt_autodiscovery-amd64
```

The add-on should continue to work as it currently does when it runs in add-on mode.
